projects/1.16/assets/just-enough-effect-descriptions-jeed/jeed/lang/zh_cn.json

```
@@ -1,8 +1,43 @@
 {
-  "jeed.category.effect_info": "效果",
-  "jeed.tooltip.color_complete": "%1$s：%2$s",
-  "jeed.tooltip.color": "颜色",
-  "jeed.tooltip.beneficial": "有益",
-  "jeed.tooltip.harmful": "有害",
-  "jeed.description.missing": "暂无描述"
+    "jeed.category.effect_info": "效果",
+    "jeed.tooltip.color_complete": "%1$s：%2$s",
+    "jeed.tooltip.color": "颜色代码",
+    "jeed.tooltip.beneficial": "增益效果",
+    "jeed.tooltip.harmful": "负面效果",
+    "jeed.description.missing": "无可用描述",
+  
+    "effect.minecraft.absorption.description": "给予吸收伤害用生命值（不会自然恢复）；等级越高，给予的生命值越高。",

```

```suggestion
    "effect.minecraft.absorption.description": "给予伤害吸收生命值（不会自然恢复）；等级越高，给予的生命值越高。",

```
来源：[伤害吸收](https://minecraft.fandom.com/zh/wiki/%E4%BC%A4%E5%AE%B3%E5%90%B8%E6%94%B6)